### DISCUSSION TITLE
kickstart proxmox

### DISCUSSION THREAD

**Discussion by andrewm659** [cobbler/cobbler#3432] [Q&A]
Has anyone been able to get proxmox added to cobbler?  I don't need a preseed file for it just boot it from the network and install it.  Would the process be similar to adding the ESXi 7.0??

**Comment by SchoolGuy** [comment]
I plan to download ESXi this weekend and add it to the signatures. During this process, I am also planning to test if it boots. If I have time left I will check if Proxmox can be made working. I don't know anyone in the community personally that atm has a setup like this working though.

**Reply by SchoolGuy** [reply]
Signatures are updated as PR #3418 was merged. I am currently downloading Proxmox. I believe that sadly I won't be able to test it today but next weekend is looking better.

**Comment by zparihar** [comment]
The Kickstart menu shows on Proxmox.
